source/overview.rst

```diff
@@ -63,6 +73,10 @@ Simulink Model
    :width: 200pt
    :align: center   
     
+.. Note:: 
+	Adam:
+	This is also a Simscape model. This is important to point out because it requires additional software license and knowledge beyond Simulink.
+
```

Comment:
Good point, that's stated in the dependencies but perhaps worth adding a NOTE here too.